AI health coach first post (#249)
NHSDigital/prevention-services-design-history: app/ai-health-coach/2025/09/index.md

---

> title: AI Health Coach discovery summary

Reviewer: You've used sentence case elsewhere?

```suggestion
title: AI health coach discovery summary
```

## Our problem statement

> Users of prevention services often experience support that feels disconnected or incomplete, and struggle with tools that only help with one aspect of their health – rather than supporting their broader, interconnected needs. While AI has the potential to improve personalisation and connection across services, it’s currently unclear how best to use AI to support across all aspects of someone’s prevention journey.

## Our users

Our users are those participating in (or could benefit from) healthy behaviour change or prevention services. Our review of existing research focused on users at various stages of that journey:

- Better Health users and NHS App users
- adults in the UK aged between 18 and 60, who were mostly from deprived areas (socioeconomic groups C2, D, E)
- people with at least one raised health risk factor (being overweight, being inactive, smoking, heavier drinking)

Other teams across NHS England and Department for Health and Social Care had already carried out extensive research to understand the pain points and barriers across this journey, so we drew upon these rather than conducting new primary research.

## Research and analysis

> We identified pain points and user needs, as well as assess how these are being met by the existing landscape. We explored:

Reviewer: This sentence feels slightly ungrammatical with a mix of tenses ('identified' and 'asses')?

Could it be something like this:

```suggestion
We identified pain points and user needs, and assessed how these are being met by the existing landscape. We explored:
```

### The existing prevention and behaviour change journey

We mapped the user journey based on what we knew about prevention and behaviour change services from our experience across NHS England and Better Health. The journey was split into 9 sections, and 4 high‑level stages:

> 1. become aware and engage
> - awareness for change
> - seek help
> 2. find and select a service
> - find a prevention service
> - select a service
> - wait for service to start
> 3. enrol and engage in a service
> - first contact and triage
> - ongoing service engagement
> - completion of service engagement
> 4. feedback and next steps
> - feedback and next steps

Reviewer: I think the sub-lists can be indented to make them slightly visually clearer? Not sure if that works or not though in Markdown.

```suggestion
1. become aware and engage
   - awareness for change
   - seek help
2. find and select a service
   - find a prevention service
   - select a service
   - wait for service to start
3. enrol and engage in a service
   - first contact and triage
   - ongoing service engagement
   - completion of service engagement
4. feedback and next steps
   - feedback and next steps
```

### Feasibility and viability of delivering this

We explored the technical design. This could use several AI models working together to provide the best answer in line with agreed behaviour change models and clinical safety parameters. We began to look at ways in which Health Coach could deal with escalations from lifestyle topics into clinical – for example by integrating with the 111 Online service. This is something that would continue to be tested in alpha.

We also considered:

- cost and benefits – the societal benefits of delivering Health Coach are likely to significantly outweigh the costs of delivering it, but not in direct cash-savings for the NHS.
- information governance and legal basis – Health Coach would likely use a consent‑based model for processing data, but there are further dependencies to explore (including where it sits and which data it uses).
- strategy and policy – Health Coach fits with health policies, with a specific mention in the 10‑Year Health Plan.

> ## Conclusion and plan for Alpha

Author: ```suggestion
## Conclusion and plan for alpha
```

> We found significant evidence to support both elements of our problem statement, and that a solution is feasible and viable. We recommended that the project progress into Alpha to test our riskiest assumptions:

Author: ```suggestion
We found significant evidence to support both elements of our problem statement, and that a solution is feasible and viable. We recommended that the project progress into alpha to test our riskiest assumptions:
```
